devzair/mail-logger-bundle/2.0/manifest.json

```
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "DevZair\\MailLoggerBundle\\MailLoggerBundle": [
+            "all"
+        ]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "requirements": {
+        "php": ">=8.1",
+        "symfony": "^6.0 || ^7.0"
+    }

```

Please remove this section. `requirements` is not a supported option for `manifest.json`.
Here is configuration reference https://github.com/symfony/recipes?tab=readme-ov-file#full-example
```suggestion
    }
```